frontend/components/atoms/buttons/button.php

```diff
@@ -1,35 +1,21 @@
 <?php
 
-$class  = $args['class'] ?? '';
-$label  = $args['label'] ?? 'press me';
-$url    = $args['url'] ?? '';
-$target = $args['target'] ?? '';
+switch ( $args['button_style'] ?? 'default' ) {
+	case 'link':
+		$class = 'text-regular text-link-primary cursor-pointer underline hover:no-underline inline-block';
+		break;
+	default:
+		$class = 'text-regular text-primary-btn bg-primary-btn-bg text-link-primary cursor-pointer px-primary-btn-padding-x py-primary-btn-padding-y hover:opacity-90 inline-block';
+		break;
+}
 
-$target = empty( $target ) ? '_self' : $target;
-
-$class .= ' inline-block py-2 px-2 font-extrabold text-primary border-2 transition-2s ';
 ?>
 
-<?php if ( $url ) : ?>
-
-	<a
-		data-atom="buttons/button"
-		class="<?php echo esc_attr( $class ); ?>"
-		href="<?php echo esc_url( $url ); ?>"
-		target="<?php echo esc_attr( $target ); ?>">
-
-		<?php echo esc_html( $label ); ?>
-
-	</a>
-
-<?php else : ?>
-
-	<button
```

Comment:
this button tag is useful when needing triggering something without link

Reply:
ok, i see. let me look into this.

Reply:
Updated